pyproject.toml

```diff
@@ -46,6 +46,7 @@ requests = "^2.25.1"
 [tool.poetry.dev-dependencies]
 
 # test dependencies
+lxml = ">=4.6.3"
```

Comment:
Please use version selection as described here:
https://python-poetry.org/docs/dependency-specification/

In this case ``^4.6.3`` would be good

Reply:
done